lib/health_web/controllers/exercise_controller.ex

```diff
@@ -0,0 +1,60 @@
+defmodule HealthWeb.ExerciseController do
+  use HealthWeb, :controller
+  alias Health.Users.User
+  alias Health.Exercise
+  alias Plug.Conn
+
+  action_fallback HealthWeb.FallbackController
+
+  @spec index(%Conn{assigns: %{current_user: %User{}}}, any) :: {:error, any} | %Conn{}
+  def index(conn, _params) do
+    user = get_current_user(conn)
+
+    exercises = Exercise.all()
+
+    with :ok <- Bodyguard.permit(Health.Policies.Exercise, :index, user, Exercise) do
+      render(conn, "index.html", exercises: exercises)
+    end
+  end
+
+  @spec new(%Conn{assigns: %{current_user: %User{}}}, any) :: {:error, any} | %Conn{}
+  def new(conn, _params) do
+    user = get_current_user(conn)
+
+    exercise = %Exercise{}
+    changeset = Exercise.changeset(exercise, %{})
+
+    with :ok <- Bodyguard.permit(Health.Policies.Exercise, :new, user, exercise) do
+      render(conn, "new.html", exercise: exercise, changeset: changeset)
+    end
+  end
+
+  @spec create(%Conn{assigns: %{current_user: %User{}}}, %{exercise: map}) :: {:error, any} | %Conn{}
+  def create(conn, %{exercise: params}) do
+    user = get_current_user(conn)
+
+    exercise = %Exercise{}
+
+    with :ok <- Bodyguard.permit(Health.Policies.Exercise, :create, user, exercise) do
```

Comment:
alias `Health.Policies.Exercise`

Reply:
Ohhh.. you have a namespacing issue

Reply:
Depending on how we want to implement Policies. They could possibly be a top level idea and should get their own namespace.